SPECS/kata-containers/drop-mut-for-variables-that-are-not-mutated.patch

```diff
@@ -0,0 +1,54 @@
+From 19a8a137b1c5fd9248896bd5f63638acfc9aff8c Mon Sep 17 00:00:00 2001
+From: Muhammad Falak R Wani <falakreyaz@gmail.com>
```

Comment:
@mfrw please post this patch into upstream kata-containers as well so upstream gets the fix

Reply:
@christopherco  Upstream already has these fixes; i've sent the relevant fixes to our kata fork though.